lightning-block-sync/src/poll.rs

```diff
@@ -1,3 +1,5 @@
+//! Adapters which make one or more [`BlockSource`]s simpler to poll for new chain tip transitions.
```

Comment:
Will this work given the explanation for using the `crate::` prefix in 1e71d32a996c97b49d4c828325705d6234413793? Likewise in other files.

Reply:
As long as its imported in a use statement later in the file it seems to work (cargo doc doesn't error, which it now will if it doesnt resolve).